org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/PureXbaseUiModule.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * generated by Xtext
+ */
+package org.eclipse.xtext.purexbase.ui;
+
+import java.util.Collection;
+import java.util.Collections;
+
+import org.eclipse.ui.plugin.AbstractUIPlugin;
+import org.eclipse.xtext.purexbase.ui.autoedit.AutoEditStrategyProvider;
+import org.eclipse.xtext.ui.editor.autoedit.AbstractEditStrategyProvider;
+import org.eclipse.xtext.ui.editor.folding.FoldedPosition;
+import org.eclipse.xtext.ui.editor.folding.IFoldingRegionProvider;
+import org.eclipse.xtext.ui.editor.model.IXtextDocument;
+
+/**
+ * Use this class to register components to be used within the Eclipse IDE.
+ */
+public class PureXbaseUiModule extends AbstractPureXbaseUiModule {
+
+	public static class NoFolding implements IFoldingRegionProvider {
+		@Override
+		public Collection<FoldedPosition> getFoldingRegions(IXtextDocument xtextDocument) {
+			return Collections.<FoldedPosition> emptySet();
```

Comment:
static import? 
`emptySet()` should be enough

Reply:
This is just cosmetics. I would be fine with leaving this. Either change it or leave it.